app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def three_most_recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] Kindly move this method to the user model, you are supposed to add a method that updates the likes counter for a post.

Reply:
Well resolved 👍🏽